src/main/java/de/muenchen/isi/api/controller/AbfrageStatusController.java

```diff
@@ -32,16 +32,155 @@ public class AbfrageStatusController {
 
     @PutMapping("infrastruktur-abfrage/{id}/freigabe")
     @Transactional
-    @Operation(summary = "Gibt eine Infrastrukturabfrage frei")
+    @Operation(summary = "Setzt eine Infrastrukturabfrage auf den Status OFFEN")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "OK -> Abfrage wurde erfolgreich freigegeben."),
             @ApiResponse(responseCode = "404", description = "NOT_FOUND -> Es gibt keine Abfrage mit der ID.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class))),
             @ApiResponse(responseCode = "409", description = "CONFLICT -> Die Abfrage konnte aufgrund des aktuellen Status nicht freigegeben werden.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class)))
     })
     @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_FREIGABE_ABFRAGE.name())")
     public ResponseEntity<Void> freigabeInfrastrukturabfrage(@PathVariable @NotNull final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
-        this.abfrageStatusService.freigabeInfrastrukturabfrage(id);
+        this.abfrageStatusService.freigabeAbfrage(id);
         return ResponseEntity.ok().build();
     }
 
+    @PutMapping("infrastruktur-abfrage/{id}/abbrechen")
+    @Transactional
+    @Operation(summary = "Setzt eine Infrastrukturabfrage auf den Status ABBRUCH")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "OK -> Abfrage wurde erfolgreich abbgebrochen."),
+            @ApiResponse(responseCode = "404", description = "NOT_FOUND -> Es gibt keine Abfrage mit der ID.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class))),
+            @ApiResponse(responseCode = "409", description = "CONFLICT -> Die Abfrage konnte aufgrund des aktuellen Status nicht abbgebrochen werden.", content = @Content(schema = @Schema(implementation = InformationResponseDto.class)))
+    })
+    @PreAuthorize("hasAuthority(T(de.muenchen.isi.security.AuthoritiesEnum).ISI_BACKEND_ABBRECHEN_ABFRAGE.name())")
+    public ResponseEntity<Void> abbrechenInfrastrukturabfrage(@PathVariable @NotNull final UUID id) throws EntityNotFoundException, AbfrageStatusNotAllowedException {
+        this.abfrageStatusService.abbrechenAbfrage(id);
+        return ResponseEntity.ok().build();
+    }
+
+    @PutMapping("infrastruktur-abfrage/{id}/angabe-anpassen")
```

Comment:
Kann man nicht über die Klasse die Annotation schreiben 
`@RequestMapping(value = CONTROLLER_PATH)` um doppelten Code und Tippfehler zu vermeiden.

Als Konstante setzen:
`public static final String CONTROLLER_PATH = "infrastruktur-abfrage";`

```suggestion
    @PutMapping("/{id}/angabe-anpassen")
```